docs/source/releasenotes/add-ons/3.0.rst

```diff
@@ -0,0 +1,42 @@
+.. _Version30ReleaseNotes:
+
+3.0 Release Notes
+===================
+
+Move koverse-addons to separate repo in Github
+
+
+Dependency upgrades
+-------------------
+
+3.0.4
+^^^^^
+
+changed koverse-sdk version to 3.4.1
```

Comment:
Not sure that sdk version is necessary in the release notes. That is more of an implementation detail.

Reply:
removed the koverse-sdk version